API.md

```diff
@@ -131,6 +132,17 @@ Content-Type: application/json; charset=UTF-8
 
 ```
 
+#### Request Example (set power)
+```javascript
+PUT /v1/pen
+Content-Type: application/json; charset=UTF-8
+
+{
+    "power": 0.75  // Pen power is a float from 0 (Full off) to 1 (Full on)
```

Comment:
For the sake of completeness here, it would be exceedingly nice for us to include a comment in the usage notes below explaining what "power" actually sets, and what it actually does on the EBB.